manage.py

```diff
@@ -87,6 +87,40 @@ def set_default_settings_path(argv):
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", module)
 
 
+def set_nosetests_verbosity(argv):
+    """Increase nose output verbosity with -v... argument
+
+        -v: print test names
+        -vv: do not capture stdout
+        -vvv: do not capture logging
+        -vvvv: enable nose internal logging
+    """
+    import logging
+
+    def set_verbosity(arg, i):
+        args = []
+        verbosity = sum(1 for c in arg if c == "v") + 1
+        if len(arg) > verbosity:
+            # preserve other single-letter arguments (ex: -xv)
```

Comment:
This seems like it shouldn't need to be done manually - is there any way we could use `argparse` or similar to handle this sort of thing?

Reply:
argparse is used internally by django and nose. I'm not sure there's a good way to use it to parse, mutate, and re-constitute a set of arguments that would be easier than what this code is doing.